visual-diff.js

```diff
@@ -44,8 +45,17 @@ after(async() => {
 		process.stdout.write('Stopped server.\n');
 	}
 	process.stdout.write(chalk.green(`\n  ${chalk.green(_goldenUpdateCount)} golden(s) updated.\n`));
+	if (_goldenOrphanCount > 0) {
+		process.stdout.write(chalk.green(`\n  ${chalk.green(_goldenOrphanCount)} golden(s) orphaned.\n`));
+	}
 	if (_goldenErrorCount > 0) {
-		process.stdout.write(chalk.red(`  ${chalk.red(_goldenErrorCount)} golden updates failed.\n`));
+		process.stdout.write(chalk.red(`\n  ${chalk.red(_goldenErrorCount)} golden updates failed.\n`));
+	}
+	process.stdout.write('\n');
+
+	if (_goldenUpdateCount === 0 && _goldenOrphanCount > 0) {
+		// force return code if no tests are triggering updates, otherwise PR will not be opened to remove orhpahned goldens
```

Comment:
```suggestion
		// force return code if no tests are triggering updates, otherwise PR will not be opened to remove orphaned goldens
```
This library doesn't _technically_ know it's being used in a workflow that opens a PR, but they're tightly coupled anyways